php/libraries/Database.class.inc

```diff
@@ -974,9 +974,11 @@ class Database
         if ($params) {
             $query = str_replace(array_keys($params), array_values($params), $query);
         }
-        print "$this->_databaseName:"
+        $query = json_encode($query);
+        print "console.log('"
```

Comment:
@alisterdev @driusan  say use: console.info instead so that Travis does not fail